src/debug/args-test.js

```diff
@@ -86,6 +87,7 @@ class ArgsTest {
         const breakpointFile = path.join(engine.cwd, this.sourcePath, 'ArgsTest.java');
         const expectedLine = 20;
         const outputList = [];
+        var assertCount=0;
```

Comment:
var => let

let assertCount = 0; // be careful about the code style.

Reply:
Got it